### PR TITLE
Pass locale correctly to Gregorian timestamps in Urdu 

### LINKED ISSUE #2822: Pass locale correctly to Gregorian timestamps in Urdu.
**Is your feature request related to a problem? Please describe.**
We need to update our Gregorian timestamps to support Arabic, Urdu and Pashto, this includes using different translations for each, **different numerals for Pashto**, and right-to-left support for all.

We already completed this work for Persian in https://github.com/bbc/simorgh/issues/1969 so this issue should be extending that further to encompass the requirements for the other 3 services in this group.

**Describe the solution you'd like**

When viewing an article in Urdu, the timestamp should:
Use Western Arabic numerals (i.e. 12345678...) (possibly returned when using moment's ur locale) 
Be right-to-left
Have onscreen and offscreen translations in Urdu for Updated text
When time is updated, the updated timestamps should all read in Urdu

**Describe alternatives you've considered**

**Testing notes**
We will want to test that the timestamp returns as expected depending on the locale that's being passed through. We may also want to investigate if there are any changes we want to make to the Cypress tests given we're introducing extra logic to the timestamp.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by RayNjeri**
Resolves #2822

**Overall change:** _Pass locale correctly to Gregorian timestamps in Arabic & Urdu services._

**Code changes:**

- _Imported respective moment locales to the services for the timestamp translations._
---
For Urdu:
Screen Shot 2019-08-14 at 1 05 13 PM

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by HarveyPeachey — APPROVED** [review]
LGTM

File: cypress/integration/pages/articles.js [hunk 1]
```diff
@@ -11,6 +12,7 @@ const serviceHasFigure = service =>
 const serviceHasCaption = service => service === 'news';
 // TODO: Remove after https://github.com/bbc/simorgh/issues/2962
 const serviceHasCorrectlyRenderedParagraphs = service => service !== 'sinhala';
+const serviceHasTimestamp = service => service === 'news';
```
Comment (rhenshaw56) [inline cypress/integration/pages/articles.js:15]: can't we re-use `serviceHasCaption` ? seems to do the same thing

Comment (RayNjeri) [reply cypress/integration/pages/articles.js:15]: `serviceHasTimestamp` looks for the timestamp on articlePage
`serviceHasCaption` looks for image/video captions

**Review by rhenshaw56 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**